src/components/plugins/AccountComponent.js

```diff
@@ -0,0 +1,63 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import Result from './ResultComponent';
+  
+class AccountComponent extends Component {
+
+    getCloudScore(results) {
+        let res = 0;
+        for (let i = 0; i < results.length; i++) {
+            const element = results[i];
+            res += ((element.passed * 100) / element.checked);
```

Comment:
You don't handle errors in result (If `element.checked` is 0, res will be `NaN`).